src/main/pages/setup-docker/docker-config.adoc

```diff
@@ -472,56 +497,68 @@ CHE_HTTPS_PROXY_FOR_WORKSPACES=<YOUR_PROXY_FROM_DOCKER>
+
+. In the left pane, right-click *Inbound Rules*, and then click *New Rule* in the action pane.
+
+. In the *Rule Type* dialog box, select *Port*, and click *Next*.
 
-*Running Che Behind a Firewall (Windows)*
+. In the *Protocol and Ports* dialog box, select *TCP*.
 
-There are many third party firewall services. Different versions of Windows OS also have different firewall configurations. The built-in Windows firewall can be configured in the control panel under "System and Security":
+. Select specific local ports, enter the port number to be opened, and click *Next*.
 
-1.  In the left pane, right-click `Inbound Rules`, and then click `New Rule` in the action pane.
-2.  In the `Rule Type` dialog box, select `Port`, and then click `Next`.
-3.  In the `Protocol and Ports` dialog box, select `TCP`.
-4.  Select specific local ports, enter the port number to be opened and click `Next`.
-5.  In the `Action` dialog box, select `Allow the Connection`, and then click `Next`.
-6.  In the `Name` dialog box, type a name and description for this rule, and then click `Finish`.
+. In the *Action* dialog box, select *Allow the Connection*, and click *Next*.
 
-*Limiting Che Ports*
+. In the *Name* dialog box, type a name and description for this rule, and click *Finish*.
 
-Eclipse Che uses Docker to power its workspaces. Docker uses the https://en.wikipedia.org/wiki/Ephemeral_port[ephemeral port range] when exposing ports for services running in the container. So when a Tomcat server is started on port 8080 inside a Che workspace Docker automatically selects an available port from the ephemeral range at runtime to map to that Tomcat instance.
+*Limiting Che ports*
 
-Docker will select its ports from anywhere in the ephemeral range. If you wish to reduce the size of the ephemeral range in order to improve security you can do so, however, keep in mind that each Che workspace will use at least 2 ports plus whatever ports are required for the services the user adds to their workspace.
+Eclipse Che uses Docker to power its workspaces. Docker uses the https://en.wikipedia.org/wiki/Ephemeral_port[ephemeral port range] when exposing ports for services running in the container. So when a Tomcat server is started on port `8080` inside a Che workspace, Docker automatically selects an available port from the ephemeral range at runtime to map to that Tomcat instance.
 
-Limiting the ephemeral range can only be done at the host level - you can read more about it (and some of the risks in doing so) here: http://www.ncftp.com/ncftpd/doc/misc/ephemeral_ports.html
+Docker will select its ports from anywhere in the ephemeral range. If you wish to reduce the size of the ephemeral range in order to improve security you can do so. However, you must note that each Che workspace will use at least 2 ports and, additionally, ports are required for the services that the user adds to their workspace.
```

Comment:
no if, no however

Reply:
Ad 'however': Not sure who said that to you, but words like 'however' in this context are redundant. If 'however' is included in the middle of a sentence, then it can serve a purpose as a conjunction. Here, it is just noise. Compare:

_However, you must note that..._

with:

_Note that..._

Same goes for the 'if' in the preceding sentence. Compare:

_If you wish to reduce..._

with:

_To reduce..._

For further reference, look at, for example, https://www.tcbok.org/wiki/minimalism/, which provides useful tips, such as:

* Clear, concise text with a minimum of extra words is most helpful to users. 
* Use short sentences, simple sentence structures, and simple words. Avoid extra words.

---------------------

Ad 'two ports' -- yes, you're right.

Reply:
I agree with all that you said above. Maybe, next time when I come across one, I'll point it out to you. For now, I have all your suggestions incorporated. Thanks!